macOS-HS-libvirt.xml

```diff
@@ -159,11 +152,10 @@
     </hub>
     <memballoon model='none'/>
   </devices>
+  <!-- Note: Remove the next line when SELinux is not installed -->
   <seclabel type='dynamic' model='selinux' relabel='yes'/>
   <qemu:commandline>
     <qemu:arg value='-device'/>
     <qemu:arg value='isa-applesmc,osk=ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc'/>
-    <qemu:arg value='-smbios'/>
```

Comment:
Is this removal strictly required? I am not sure about this.

Reply:
This was added in https://github.com/kholia/OSX-KVM/pull/93 to resolve incorrect MacOSX performance which is properly resolved with TSC. I can readd it, but I always ran MacOSX without this line and never encountered issues.